content/articles/getting-started-with-python-plug-ins-for-gimp/index.md

```diff
@@ -0,0 +1,128 @@
+### Introduction 
+GIMP is an open-source image manipulation tool which many people use as a suitable solution as it competes with some corporate demands. GIMP is efficient in dealing with tangled elements such as layers and paths. GIMP is aware of many coordinated picture associations and works with channels that are typically elaborate. GIMP has a solid foundation and philosophy, thus there isn't any technical info as to how to use or encourage GIMP in general.

```

Comment:
This sentence is too long. What are tangled elements? 

When you say `GIMP is aware of many coordinated picture associations and works with channels that are typically elaborate. GIMP has a solid foundation and philosophy, thus there isn't any technical info as to how to use or encourage GIMP in general.` what do you mean?